packages/contracts-bedrock/scripts/upgrades/holocene/README.md

```diff
@@ -0,0 +1,28 @@
+# Holocene Upgrade
+
+This directory contains a repeatable task for upgrading an `op-contracts/v1.6.0` deployment to `op-contracts/v1.8.0`.
+
+## Dependencies
+
+- `docker`
+- `just`
+- `foundry`
```

Comment:
Any specific versions or install commands/URLs we should provide?

Reply:
No specific versions required. Will add links.